app/api/helpers/tasks.py

```diff
@@ -663,3 +662,15 @@ def delete_translations(self, zip_file_path):
         os.remove(zip_file_path)
     except:
         logger.exception('Error while deleting translations zip file')
+
+
+@celery.task(name='rename.chat.room')
+def rename_chat_room(event_id):
+    event = db.session.query(Event).filter_by(id=int(event_id)).first()
+    try:
+        rename_rocketchat_room(event=event)
+        logging.info("Rocket chat room renamed successfully")
+    except Exception as e:
```

Comment:
Don't catch bare exception

Reply:
I dont understand, what is supposed to be done here?

Reply:
Nothing, remove try catch